docs/main/user-guides/upgrade.md

```diff
@@ -20,57 +20,139 @@ Upgrade to the latest cStor or Jiva version is supported only from 1.12.0 or lat
+You'll need the helm v3 binary to be on you $PATH environment variable. Execute the following command to check if you have helm v3:
 ```
-kubectl get node -l openebs.io/engine=mayastor
+$ helm version --short
+v3.9.4+gdbc6d8e
 ```
 
-From the list of nodes obtained using the above command, select a node and perform the Mayastor node-drain operation on it.
+Execute the following command to list the helm releases on the namespace where Mayastor is installed:
 ```
-kubectl mayastor drain node <node-name> <label>
+$ helm list -n openebs
+NAME            	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART        	APP VERSION
+openebs-mayastor	openebs  	1       	2023-05-26 22:03:15.473035 +0530 IST	deployed	openebs-3.6.0	3.6.0   
 ```
-:::note
-Add a custom drain-label to the above drain process in order to ensure no conflicts with any other node-drain/node-cordon labels. 
-:::
+If the value of the 'CHART' column in the row which contains the helm chart for Mayastor, in the output of the above command, reads 'openebs-...', then your Mayastor deployment uses the openebs helm chart. If the CHART column instead reads 'mayastor-...', then it makes use of the mayastor helm chart.
+[Click here if your Mayastor deployment uses the 'mayastor' helm chart](https://mayastor.gitbook.io/introduction/additional-information/upgrade).
+
+Continue down this page if your Mayastor deployment uses the 'openebs' helm chart.
+
+### Upgrading Mayastor installed from the openebs helm chart
+
+#### 1. Planning the upgrade
+Supported upgrade paths for upgrade for Mayastor, installed with the openebs chart, are...
+Source versions: 3.4.x or above
+Target versions: 3.7.y or above
+All source versions may upgrade to any of the target versions, so long as the release semver of the target version is greater than that of the source version.
+
+To check for the latest released version of Mayastor available with the openebs helm chart, execute the following commands -- `helm repo update`, `helm search repo openebs --version ">a.b.c"`, where a.b.c is your source version, i.e. the version installed on your cluster. E.g.:
+```
+$ helm repo update openebs             
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "openebs" chart repository
+Update Complete. ⎈Happy Helming!⎈
+
+$ helm search repo openebs --version ">3.5.0"
+NAME           	CHART VERSION	APP VERSION	DESCRIPTION                                  
+openebs/openebs	3.7.0        	3.7.0      	Containerized Attached Storage for Kubernetes
+```
+
+If you want to list all of the versions available for upgrade, execute the following commands:
+```
+$ helm repo update openebs             
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "openebs" chart repository
+Update Complete. ⎈Happy Helming!⎈
+
+$ # Assuming source version of 3.5.0
+$ helm search repo openebs --version ">3.5.0,>=3.7.0" -l
```

Comment:
Should it be <=3.7.0 ?

Reply:
This is trying to list version above the user's semver version, and also above the semver version from which upgrades are supported.